components/signersForm.tsx

```diff
@@ -576,6 +687,16 @@ const SignersForm: FC<{
                   )
                 : null}
             </div>
+            <div className="mt-4 w-full">
+              <label className="block text-white">Delegate wallet</label>
```

Comment:
I am thinking maybe the title should be like  "Change wallet participants below" in line 489 so as the "Threshold" and "Proposal duration". what do you think?